var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -114,6 +114,9 @@ class Openblas(CMakePackage, MakefilePackage):
     # https://github.com/OpenMathLib/OpenBLAS/pull/4328
     patch("xcode15-fortran.patch", when="@0.3.25 %apple-clang@15:")
 
+    # https://github.com/OpenMathLib/OpenBLAS/pull/5207
+    patch("xcode16-fortran.patch", when="@0.3.29")
```

Comment:
I have the following in my local checkout if you want to steal it:
```python
    # https://github.com/OpenMathLib/OpenBLAS/issues/5202                                           
    patch(                                                                                          
        "https://github.com/OpenMathLib/OpenBLAS/pull/5207.patch?full_index=1",                     
        sha256="3b5041a32732a108a071058d018f82d233d0cd3216ffb96bed29ca409b037a0f",                  
        when="@0.3.29",                                                                            
    )  
```
Then we don't have to include the file itself.

P.S. Can you check if this patch applies to 0.3.28 and older?

Reply:
I usually just do a binary search and see how far back I can go. If you don't know about it, `spack patch openblas@0.3.27` is very useful to quickly test.

Reply:
@adamjstewart Thanks! Look like 0.3.27 is the farthest back we can go. So I think I've fixed this up the right way.